src/TraceEvent/TraceEvent.cs

```diff
@@ -186,6 +200,20 @@ public KernelTraceEventParser Kernel
             }
         }
 
+        /// <summary>
+        /// For convenience, we provide a property returns a InteropTraceEventParser that knows 
+        /// how to parse the Interop events into callbacks.
+        /// </summary>
+        public InteropTraceEventParser Interop
```

Comment:
Same as above.   I don't want TraceEvent (which is low in the dependency hierarchy) to take a dependency on basically 'random' parsers (which are high in the dependency hierarchy).  

Just new it up as you need it.   And remove it from here.

Reply:
Got you